packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -1121,11 +1121,15 @@ class RenderListWheelViewport
   }
 
   @override
-  RevealedOffset getOffsetToReveal(RenderObject target, double alignment, { Rect? rect }) {
+  RevealedOffset getOffsetToReveal(
+    RenderObject target,
+    double alignment, {
+    Rect? rect,
+    Axis? axis,
```

Comment:
or why not make this required?

Reply:
is this tristate? if so we should either document what null means, especially when TwoD case. Can also consider turn this into an enum to be more readable

Reply:
It's not required. It is documented in the super class.